### DISCUSSION TITLE
Use variables in function

### DISCUSSION THREAD

**Discussion by MarcelWeidum** [alpinejs/alpine#1043] [1. Help]
```
<template x-for="[id, child] in Object.entries(variant.variants)">
     <a x-on:click="fetchVariants({ id: child })" x-text="child" href="#"></a>
</template>
```

Does someone know how I use the 'id' and the 'child' variable in the fetchVariants() function?

**Comment by ekwoka** [comment]
Yeah, just pass them to it.

Here you are passing an object of `{ id: VALUE OF CHILD VARIABLE }`. to do `{ [VALUE OF ID VARIABLE]: VALUE OF CHILD VARIABLE }` it should be `{ [id]: child }`. I'd probably just do `fetchVariants(id, child)` or even just put the `id` on the actual child object in the original data.
